### DISCUSSION TITLE
VS code failed start erlang_ls when using remote-SSH

### DISCUSSION THREAD

**Discussion by trjsw** [erlang-ls/erlang_ls#1377] [General]
Hi,
I am trying to start erlang_ls from my local computer which uses remote-SSH connect to a server. But it failed with an error message like the below: 
[Error - 12:02:02 PM] Starting client failed
Launching server using command escript failed.

Does anyone know what the reason for it is? It is because erlang_ls hasn't supported this scenario for vscode or needs some setup for this scenario.

**Comment by Whaileee** [comment]
Actually I'm using it the way you want to. I have installed Erlang_ls from extensions, on remote, and things just worked. Maybe there is something wrong with your escrpit or erlang_ls.config

**Reply by trjsw** [reply]
Yes, the issue is escript hasn't started. It has been solved now. Thanks!
